dg/user-workflow.md

```diff
@@ -9,3 +9,14 @@
 # User workflow
 
 <!-- Add relevant content -->
+## Overview
+
+![](../images/user-workflow.png) {.center}
+
+Here is a brief explanation of each of its main features
+1. **Issue viewer**
+   1. Users will first see the issue viewer when watching a github repo with WATcher. 
+   1. They can sort and filter by various parameters such as titles, milestones and labels. 
```

Comment:
The sub-bullet points seem to be wrongly numbered

Reply:
I numbered them this way following the convention of CATcher docs which writes all bullet points as 1., letting markbind handle the appropriate numbering.
If this is unintentional it should be made an issue in CATcher docs.
![image](https://github.com/CATcher-org/WATcher-docs/assets/88131400/1ce3201d-fed8-4fe2-968d-1ab1e722d4ef)

Reply:
For now I have fixed the numbering.